src/Array/Extra.elm

```diff
@@ -233,6 +233,19 @@ zip3 =
     map3 (\a b c -> ( a, b, c ))
 
 
+{-| Unzip array of tuples into a tuple containing two arrays
+    unzip [ (1, 'a'), (2, 'b'), (3, 'c') ] == ([ 1, 2, 3 ], [ 'a', 'b', 'c' ])
```

Comment:
Could you add a newline between the description and example?

Also, when trying this out, I had to coerce the list-literal to an Array using `Array.fromList` like this

```
unzip <| fromList [ (1, 'a'), (2, 'b'), (3, 'c') ]
```

I'm not sure, if this is a recent change in Elm, as all the other examples in that file assume that you can write array literals using square brackets.

In any case, could you update the example to convert the List to Array just so the example works as typed?

Reply:
I fixed the issues you mentioned.